doc/installation.rst

```diff
@@ -0,0 +1,3 @@
+.. include:: ../README.rst
```

Comment:
@nathanshammah is this the idea we wanted to port from your PR #13?

Reply:
yes, the way it's in #13, which is copied from the Qulacs plugin, is that there are markers in the main [`README.rst`](https://github.com/unitaryfund/pennylane-qrack/blob/master/README.rst) file, that here need to be added, see [here](https://github.com/unitaryfund/pennylane-qrack/pull/13/files).

Reply:
Fixed.